plugins/modules/azure_rm_notificationhub.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2021 Praveen Ghuge (@praveenghuge), Karl Dasan (@karldas30)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+import time
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
+__metaclass__ = type
+DOCUMENTATION = '''
+---
+module: azure_rm_notificationhub
+version_added: "0.1.2"
+short_description: Manage Notification Hub
+description:
+    - Create, update and delete instance of Notification Hub.
+options:
+    resource_groups:
+        description:
+            - Name of the resource group to which the resource belongs.
```

Comment:
Parameter types are missing in the document. Please help to add all parameter types!
```suggestion
            - Name of the resource group to which the resource belongs.
        type: str
```